src/lib/decentraland/sdk-components/transform.ts

```diff
@@ -0,0 +1,86 @@
+import { Quaternion, Vector3 } from "@babylonjs/core";
+import { ComponentType, SerDe } from "../crdt-internal/components";
+import { Entity } from "../types";
+import type { ComponentOperation } from '.'
+
+export const TRANSFORM_COMPONENT_ID = 1
+
+export type Transform = {
+  position: Vector3
+  scale: Vector3
+  rotation: Quaternion
+  parent: Entity
+}
+
+export const transformSerde: SerDe<Transform> = {
+  deserialize(buffer) {
+    return {
+      position: new Vector3(buffer.readFloat32(), buffer.readFloat32(), buffer.readFloat32()),
+      rotation: new Quaternion(buffer.readFloat32(), buffer.readFloat32(), buffer.readFloat32(), buffer.readFloat32()),
+      scale: new Vector3(buffer.readFloat32(), buffer.readFloat32(), buffer.readFloat32()),
+      parent: buffer.readUint32()
+    }
+  },
+  serialize(value, buffer) {
+    buffer.writeFloat32(value.position.x);
+    buffer.writeFloat32(value.position.y);
+    buffer.writeFloat32(value.position.z);
+    buffer.writeFloat32(value.rotation.x);
+    buffer.writeFloat32(value.rotation.y);
+    buffer.writeFloat32(value.rotation.z);
+    buffer.writeFloat32(value.rotation.w);
+    buffer.writeFloat32(value.scale.x);
+    buffer.writeFloat32(value.scale.y);
+    buffer.writeFloat32(value.scale.z);
+    buffer.writeUint32(value.parent || 0);
```

Comment:
_Question_
Why `|| 0` here? Isn't `0` already the default, and this field non-nullable?

Reply:
`value.parent` may be undefined